gp-includes/routes/glossary-entry.php

```diff
@@ -106,12 +106,16 @@ public function glossary_entry_add_post( $project_path, $locale_slug, $translati
 	}
 
 	public function glossary_entries_post( $project_path, $locale_slug, $translation_set_slug ) {
-		$ge_post        = gp_post( 'glossary_entry' );
+		$ge_post = gp_post( 'glossary_entry' );
+		if ( ! is_array( $ge_post ) || ! isset( $ge_post[0]['glossary_entry_id'] ) ) {
```

Comment:
The `glosary_entries_post` method is used to update an individual entry in the glossary. When I test this change and I try to update an entry, I get this error:

![image](https://github.com/GlotPress/GlotPress/assets/1667814/2e3d19f9-df12-405e-b340-91ab08bf661f)

This is because this PR tries to check the "0" element in the array, but the key in the array is the `glossary_entry_id `.  E.g.:

```
array(1) { [5]=> array(6) { ["term"]=> string(1) "&" ["part_of_speech"]=> string(4) "noun" ["translation"]=> string(1) "&" ["comment"]=> string(0) "" ["glossary_id"]=> string(1) "1" ["glossary_entry_id"]=> string(1) "5" } }
```
This change solves this problem:

```suggestion
		if ( ! is_array( $ge_post ) || ! isset( reset($ge_post)['glossary_entry_id'] ) ) {
```

Reply:
Thanks, I don't think this will work but I have committed new code that separates out the checks which should work better.